uniffi_bindgen/src/bindings/kotlin/gen_kotlin/miscellany.rs

```diff
@@ -58,16 +62,16 @@ macro_rules! impl_code_type_for_miscellany {
 
 impl_code_type_for_miscellany!(
     TimestampCodeType,
-    "Instant",
+    "java.time.Instant",
     "Timestamp",
-    vec!["java.time.Instant", "java.time.DateTimeException"],
+    vec!["java.time.DateTimeException"],
```

Comment:
It could be nice to remove this import as well, just in case the user has a `DateTimeError` struct.